docs/reference/ml/apis/stop-dfanalytics.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[testenv="platinum"]
+[[stop-dfanalytics]]
+=== Stop {dfanalytics} API
+
+[subs="attributes"]
+++++
+<titleabbrev>Stop {dfanalytics}</titleabbrev>
+++++
+
+beta[]
+
+Stops one or more {dfanalytics}.
+
+[discrete]
+[[ml-stop-dfanalytics-request]]
+==== {api-request-title}
+
+`POST _ml/data_frame/analytics/<data_frame_analytics_id>/_stop`
+
+`POST _ml/data_frame/analytics/<data_frame_analytics_id>,<data_frame_analytics_id>/_stop`
+
+`POST _ml/data_frame/analytics/_all/_stop`
+
+[discrete]
+[[ml-stop-dfanalytics-prereq]]
+==== {api-prereq-title}
+
+You must have `machine_learning_admin` built-in role to use this API. For more 
+information, see {stack-ov}/security-privileges.html[Security privileges] and 
+{stack-ov}/built-in-roles.html[Built-in roles].
+
+[discrete]
+[[ml-stop-dfanalytics-path-params]]
+==== {api-path-parms-title}
+
+`<data_frame_analytics_id>` (required)::
+  (string) Identifier for the {dfanalytics} job. This identifier can contain
```

Comment:
This implies it's stopping a job. If that's correct, I think the API should have that in its name too (i.e. "stop data frame analytics jobs"). Though it begs the question of why we "stop" these jobs but "close" anomaly detector jobs. I notice we talk about "stopping a running data frame analytics _config_" in https://www.elastic.co/guide/en/elasticsearch/client/java-rest/master/java-rest-high-x-pack-ml-stop-data-frame-analytics.html
If these are indeed "jobs" I think we should be consistent in all places.

Reply:
After discussing the terminology, I add `job` to the name of the APIs where it is appropriate. However, I'd keep start/stop, because the API uses `/_start` and `/_stop` in the request while the anomaly detectors use `/_open` and `/_close`. This way, the df analytics terminology remains consistent with its own API request naming convention. What do you think?